features/logging/collector.feature

```diff
@@ -211,3 +211,41 @@ Feature: collector related tests
     Given evaluation of `JSON.parse(@result[:response])['aggregations']['exists_field_systemd']['distinct_node_ip']['buckets'].map {|furn| furn["key"]}` is stored in the :journal_ips clipboard
     And the expression should be true> Set.new(cb.node_ips) == Set.new(cb.journal_ips)
     """
+
+  # @author qitang@redhat.com
+  # @case_id OCP-32197
+  @admin
+  @destructive
+  Scenario: Fluentd should write it's own logs to stdout and exclude them from collection
+    Given I obtain test data file "logging/clusterlogging/example.yaml"
+    When I create clusterlogging instance with:
+      | remove_logging_pods | true         |
+      | crd_yaml            | example.yaml |
+    Then the step should succeed
+    # delete ES pod to make fluentd pod generate logs
+    When I run the :delete client command with:
+      | object_type | pod                 |
+      | l           | es-node-master=true |
+    Then the step should succeed
+    And I wait until ES cluster is ready
+    And I wait for the project "openshift-logging" logs to appear in the ES pod
+    When I run the :logs client command with:
+      | resource_name | <%= cluster_logging('instance').fluentd_ready_pods.first.name %> |
+    Then the step should succeed
+    And the output should not contain:
+      | Fluentd logs have been redirected to: /var/log/fluentd/fluentd.log |
+      | If you want to print out the logs, use command:                    |
+      | oc exec <pod_name> /usr/local/logs                                 |
+    When I run the :exec client command with:
+      | pod              | <%= cluster_logging('instance').fluentd_ready_pods.first.name %> |
+      | oc_opts_end      |                                                                  |
+      | exec_command     | ls                                                               |
+      | exec_command_arg |  /var/log/                                                       |
```

Comment:
Perhaps
```
  When I execute on the "<%= cluster_logging('instance').fluentd_ready_pods.first.name %>" pod:
    | ls | /var/log/ |
```
Of course, the current one also works.

Reply:
I had hit some error when there didn't have `oc_opts_end`:
```
STDERR:
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl kubectl exec [POD] -- [COMMAND] instead.
```